build/charts/antrea/templates/simulator/statefulset.yaml

```diff
@@ -68,29 +71,24 @@ spec:
         - name: logs-volume
           mountPath: /var/log
       - name: hollow-kubelet
-        image: projects.registry.vmware.com/antrea/kubemark:v1.18.4
+        image: projects.registry.vmware.com/antrea/kubemark:v1.29.0
```

Comment:
@antoninbas the kubemark image in the dockerhub is 3 years old. How do we update this image? Push to the dockerhub manually?

Reply:
done

Reply:
You could create an "on-demand" Github workflow (`workflow_dispatch`) if you wanted to automate this process for future upgrades. It could even go into the https://github.com/antrea-io/image-utils repository instead of this one.